src/style.css

```diff
@@ -0,0 +1,85 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+.bigBox {
+  position: relative;
+  height: 70px;
+  margin-top: 20px;
+}
+
+.bigIcon {
+  border: none;
+  background-color: #fff;
+  top: 25px;
+  left: 26px;
+  position: absolute;
+  color: #687c94;
+  height: 19px;
+  width: 19px;
+}
```

Comment:
2 point in checklist
this class is very similar to smallIcon, create one base class, and in second one just override some styles that you need